src/test/java/com/aspose/imaging/cloud/test/api/BmpApiTests.java

```diff
@@ -1,7 +1,7 @@
 /*
 * --------------------------------------------------------------------------------------------------------------------
 * <copyright company="Aspose" file="BmpApiTests.java">
-*   Copyright (c) 2018 Aspose Pty Ltd.
+*   Copyright (c) 2018 Aspose.Imaging for Cloud
 * </copyright>
```

Comment:
Please use "Copyright (c) 2018 Aspose Pty Ltd. All rights" reserved in this and other cases.

Reply:
@aifeigin thanks for pointing this out.